crates/marine-macro-impl/src/parse_macro_input/item_foreign_mod.rs

```diff
@@ -52,45 +53,67 @@ fn check_foreign_section(foreign_mod: &syn::ItemForeignMod) -> Result<()> {
 
 /// Tries to find and parse wasm module name from
```

Comment:
Let's update a comment here b/c the function now parses both `module_import` and the new macro `host_import`